index.html

```diff
@@ -184,46 +199,52 @@ <h6 class="marketers__section-title">Custom Design designers</h6>
 
       <div class="marketers__section-image"></div>
     </section>
+  </div>
 
+  <div class="strategies__section-container">
     <section class="strategies__section">
       <div class="strategies__section-title-block">
-        <h2 class="strategies__section-title">Marketing Strategies</h2>
+        <h3 class="strategies__section-title">Marketing Strategies</h3>
         <p class="strategies__section-description">Join 40,000+ other marketers and get proven strategies on email
           marketing</p>
       </div>
 
       <div class="strategies__section-cards-container">
+
         <article class="strategies__section-card">
           <div class="strategies__section-image-search"></div>
           <p>By <span class="strategies__section-accent">Abdullah |</span> 03 March 2019</p>
-          <h3 class="strategies__section-title">Increasing Prosperity With Positive Thinking</h3>
+          <h4 class="strategies__section-title">Increasing Prosperity With Positive Thinking</h4>
         </article>
 
         <article>
           <div class="strategies__section-image-structure"></div>
           <p>By <span class="strategies__section-accent">Abdullah |</span> 03 March 2019</p>
-          <h3 class="strategies__section-title">Motivation Is The First Step To Success</h3>
+          <h4 class="strategies__section-title">Motivation Is The First Step To Success</h4>
         </article>
 
         <article>
           <div class="strategies__section-image-time"></div>
           <p>By <span class="strategies__section-accent">Abdullah |</span> 03 March 2019</p>
-          <h3 class="strategies__section-title">Success Steps For Your Personal Or Business Life</h3>
+          <h4 class="strategies__section-title">Success Steps For Your Personal Or Business Life</h4>
         </article>
       </div>
       <div class=" strategies__section-carousell">
+
         <div class="circle">
         </div>
         <div class="circle">
         </div>
         <div class="circle">
         </div>
     </section>
+  </div>
 
+  <div class="footer__section-container">
```

Comment:
<footer> tag should be before this tag, if needed, you can add needed class there